src/bid/score.rs

```diff
@@ -4,37 +4,35 @@
 //
 // Copyright (c) DUSK NETWORK. All rights reserved.
 
-//! Score generation
+//! Score generation module
 
 #[cfg(feature = "canon")]
 use canonical_derive::Canon;
 
 cfg_if::cfg_if! {
-    if #[cfg(feature = "std")] {
-        use crate::errors::BlindBidError;
-        use crate::bid::Bid;
-        use dusk_jubjub::JubJubAffine;
-        use dusk_plonk::prelude::*;
+if #[cfg(feature = "std")] {
```

Comment:
You don't need `cfg_if`, yu can use a single `use` statement.

Reply:
How can I? None of the crates inside of the `cfg_if` is the same.

Reply:
So? You can still group the crates in one single `use` statement:
```rust
use {
    crate::bid::Bid,
    dusk_jubjub::JububAffine,
    dusk_plonk:prelude::*
};
```